tests/models/test_popular_in_category.py

```diff
@@ -22,69 +22,78 @@
 from rectools import Columns
 from rectools.dataset import Dataset
 from rectools.models import PopularInCategoryModel
-from tests.models.utils import assert_second_fit_refits_model
+from rectools.models.popular import Popularity
+from rectools.models.popular_in_category import MixingStrategy, RatioStrategy
+from tests.models.utils import (
+    assert_default_config_and_default_model_params_are_the_same,
+    assert_get_config_and_from_config_compatibility,
+    assert_second_fit_refits_model,
+)
 
 
-@pytest.mark.filterwarnings("ignore")
-class TestPopularInCategoryModel:
-    @pytest.fixture
-    def interactions_df(self) -> pd.DataFrame:
-        interactions_df = pd.DataFrame(
-            [
-                [70, 11, 1, "2021-11-30"],
-                [70, 12, 1, "2021-11-30"],
-                [10, 11, 1, "2021-11-30"],
-                [10, 12, 1, "2021-11-29"],
-                [10, 13, 9, "2021-11-28"],
-                [20, 11, 1, "2021-11-27"],
-                [20, 14, 2, "2021-11-26"],
-                [20, 14, 1, "2021-11-25"],
-                [20, 14, 1, "2021-11-25"],
-                [20, 14, 1, "2021-11-25"],
-                [20, 14, 1, "2021-11-25"],
-                [20, 14, 1, "2021-11-25"],
-                [30, 11, 1, "2021-11-24"],
-                [30, 12, 1, "2021-11-23"],
-                [30, 14, 1, "2021-11-23"],
-                [30, 15, 5, "2021-11-21"],
-                [30, 15, 5, "2021-11-21"],
-                [40, 11, 1, "2021-11-20"],
-                [40, 12, 1, "2021-11-19"],
-                [50, 12, 1, "2021-11-19"],
-                [60, 12, 1, "2021-11-19"],
-            ],
-            columns=Columns.Interactions,
-        )
-        return interactions_df
+@pytest.fixture(name="interactions_df")
+def custom_interactions_df() -> pd.DataFrame:
```

Comment:
Why `custom`?  
1. Why not just `interactions`?
2. This word doesn't add any sense, so no sense to use it?

Same everywhere

Reply:
It was done due to pylint error – W0621: Redefining name 'interactions_df' from outer scope (line 35) (redefined-outer-name). There is an open issue on this case – https://github.com/pylint-dev/pylint/issues/6531.
Decided to change naming to reduce confusion.